batch_job/src/work_queue_factory.c

```diff
@@ -78,6 +78,9 @@ static int tasks_per_worker = -1;
 static int autosize = 0;
 static int worker_timeout = 300;
 static int consider_capacity = 0;
+int capacity_weighted = 0;
```

Comment:
Could we move these three values inside the function they are used?

Reply:
I kept these three values outside the function for the purpose of debugging. I can get creative with logging them though to keep them all inside the same function. Thoughts?

Reply:
Please do. Switches may live outside functions for convenience, but it gets ugly when the state of a function lives outside it.